extensions/amp-pinterest/0.1/amp-pinterest.css

```diff
@@ -16,14 +16,14 @@
 
 /* round buttons */
 
-.-amp-pinterest-round {
+.i-amphtml-pinterest-round {
```

Comment:
This is technically a breaking change since we are going from public CSS classes to private. Looks like we meant for this to be private in the first place but used the wrong prefix but it's a bit too late now to changes this. Likely it won't impact many pages, if any at all, but probably the benefits don't outweigh the small risk.

Reply:
I see what happened: there's a whitelist exempting these files and the rename took the file out of the whitelist.
https://github.com/ampproject/amphtml/blob/9781b46ab3c505e1ed58bcb68bc2444f12e79778/build-system/tasks/presubmit-checks.js#L56-L59

TIL that when we changed the format, we allowed users to use the old style 🙃

Reply:
@caroqliu sorry, so we should remove the `i-amphtml-` changes and rename the file in the whitelist

Reply:
No worries, thanks for the heads up.